dev/lint-python

```diff
@@ -83,6 +84,53 @@ else
     rm "$PEP8_REPORT_PATH"
 fi
 
+#### Python Document Style Checks ####
+
+# Get PYDOCSTYLE at runtime so that we don't rely on it being installed on the build server.
+# Using pep257.py which is the single file version of pydocstyle.
+PYDOCSTYLE_VERSION="0.2.1"
```

Comment:
Btw, seems like the latest version of pydocstyle is `2.1.1`. Should we use it instead?

Reply:
As called out earlier, this was single file python doc style checker, the latest does not have single file checker that can be included.